src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -862,6 +862,45 @@ async def SendCommand(self, nodeid: int, endpoint: int, payload: ClusterObjects.
             interactionTimeoutMs=interactionTimeoutMs, busyWaitMs=busyWaitMs, suppressResponse=suppressResponse).raise_on_error()
         return await future
 
+    async def SendBatchCommands(self, nodeid: int, commands: typing.List[typing.Tuple[int, ClusterObjects.ClusterCommand, typing.Type]],
+                                timedRequestTimeoutMs: typing.Union[None, int] = None,
+                                interactionTimeoutMs: typing.Union[None, int] = None, busyWaitMs: typing.Union[None, int] = None,
+                                suppressResponse: typing.Union[None, bool] = None):
+        '''
+        Send a batch of cluster-object encapsulated commands to a node and get returned a future that can be awaited upon to receive
+        the responses. If a valid responseType is passed in, that will be used to deserialize the object. If not,
+        the type will be automatically deduced from the metadata received over the wire.
+
+        nodeId: Target's Node ID
+        commands: A list of tuples of type (endpoint, cluster-object, response-type):
+        timedWriteTimeoutMs: Timeout for a timed invoke request. Omit or set to 'None' to indicate a non-timed request.
+        interactionTimeoutMs: Overall timeout for the interaction. Omit or set to 'None' to have the SDK automatically compute the
+                              right timeout value based on transport characteristics as well as the responsiveness of the target.
+        busyWaitMs: How long to wait in ms after sending command to device before performing any other operations.
```

Comment:
Why is it called "busy" wait? When is this needed? This seems like something that should be better documented. I have never seen it used. If we are simply copying args from SendCommand, it is a good time to revisit.

Reply:
Definitely used in yamltests